stats.go

```diff
@@ -527,3 +549,7 @@ var getMetricDescriptor = func(ctx context.Context, c *monitoring.MetricClient,
 var createTimeSeries = func(ctx context.Context, c *monitoring.MetricClient, ts *monitoringpb.CreateTimeSeriesRequest) error {
 	return c.CreateTimeSeries(ctx, ts)
 }
+
+var isTypeInBuilt = func(metricType string) bool {
```

Comment:
Please make this a function. I don't know why `var createTimeSeries = func` was made so, but for this one:
```go
// builtinMetric returns true if a MetricType is a heuristically known
// built-in Stackdriver metric
func builtinMetric(metricType string) bool {
      // Add more prefixes of builtin metric metrics.
      // TODO: Provide a link to builtin metrics.
      return strings.HasPrefix(metricType, "appengine.googleapis.com")
}
```

Reply:
Actually it would be useful perhaps to use @yanweiguo advice in the issue (@yanweiguo please provide us with a link to that. Thus turn this function inside out by filtering for the known non-builtin prefixes i.e.
```go
var knownExternalMetricPrefixes = []string{
   "custom.googleapis.com/",
   "external.googleapis.com/prometheus/",
}

// builtinMetric returns true if a MetricType is a heuristically known
// built-in Stackdriver metric
func builtinMetric(metricType string) bool {
     for _, knownExternalMetric := range knownExternalMetricPrefixes {
         if strings.HasPrefix(metricType, knownExternalMetric {
             return false
         }
     }
     return false
}
```

and this way the list of bug reports won't grow as this exporter gains adoption within Google.